components/record/LangMap.vue

```diff
@@ -0,0 +1,56 @@
+/*
+ * @file Presentation of "LangMap" data type in metadata
+ * @see {@link https://pro.europeana.eu/page/europeana-rest-api#getting-started|Datatypes}
+ */
+
+<template>
+  <ul data-qa="LangMap">
```

Comment:
LangMap isn't a great data-qa attribute if we're going to be using these for feature tests as outlined in the readme on the record e2e tests. Perhaps 'translated values'?

Reply:
Went with "multi-lingual metadata".